tools/cli/commands.go

```diff
@@ -56,10 +56,13 @@ const (
 	defaultTimeFormat                = "15:04:05"   // used for converting UnixNano to string like 16:16:36 (only time)
 	defaultDateTimeFormat            = time.RFC3339 // used for converting UnixNano to string like 2018-02-15T16:16:36-08:00
 	defaultDomainRetentionDays       = 3
+	defaultContextTimeoutInSeconds   = 10
```

Comment:
let's keep default 5 sec, which is actually already very long for most api.

Reply:
OK